src/components/GuardianCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { Gray1, Gray3, Gray4, TextDark, TextLight } from '../colors';
+
+import React from 'react';
+import { rem } from 'polished';
+import styled from 'styled-components';
+
+type Props = {
+  address: string;
+  status: number;
+};
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
```

Comment:
🔪

Reply:
yeah i was thrown off by having to use align-items when flex-direction is column.  will remove this